tests/samplers/test_logprobs.py

```diff
@@ -0,0 +1,58 @@
+import pytest
+import torch
+
+from vllm import SamplingParams
+
+MODELS = ["facebook/opt-125m"]
+
+
+@pytest.mark.parametrize("model", MODELS)
+@pytest.mark.parametrize("dtype", ["half"])
+def test_get_prompt_logprobs(
+    hf_runner,
+    vllm_runner,
+    model,
+    dtype,
+    example_prompts,
+):
+    max_tokens = 5
+    hf_model = hf_runner(model, dtype=dtype)
```

Comment:
What about testing the HF model first and deleting the HF model before initializing the vLLM model? This way, we can avoid the situation that the two models co-exist.

Reply:
Fixed!